genie-web/src/main/java/com/netflix/genie/web/configs/GenieServicesAutoConfiguration.java

```diff
@@ -408,4 +418,19 @@ public JobCompletionService jobCompletionService(
             retryTemplate
         );
     }
+
+    /**
+     * Get a task executor which may be shared by different components.
+     *
+     * @return A task executor
+     */
+    @Bean
+    @Lazy
```

Comment:
I'm pretty sure we already have a `TaskExecutor` in `TaskAutoConfig` or somewhere. Why do we need another one? Also it shouldn't be lazy on the server side.

Looks like this was a copy and paste from the agent.